contracts/protocol/libraries/position/TickPosition.sol

```diff
@@ -84,6 +84,10 @@ library TickPosition {
             ) {
                 index++;
             }
+            if (_self.orderQueue[index].partialFilled != 0) {
```

Comment:
if the current index has partialFilled, the total amount not filled is just `size - partialFilled` instead of `size`. this case only happens for the first index has partialFilled. the other work like the old way